app/views/content/life-as-a-teacher/teaching-as-a-career/from-teaching-internship-to-leadership-roles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: From teaching internship to leadership roles
+title_paragraph: |-
+  <p>In Ben’s second year of teaching, he took on the role of head of physics. Only 3 years later, he was promoted to assistant head of maths.</p>
```

Comment:
It's a bit weird to say that **assistant** head of maths would be considered a **promotion** above head of physics - I'm not sure that is what the article actually implies?

```suggestion
  <p>In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and become assistant head of maths.</p>
```

(reword as needed)